build/install_cuda.sh

```diff
@@ -4,22 +4,39 @@ set -xe
 CUDA_VERSION=$1
 
 LIBCUDNN=libcudnn7
-if [ $CUDA_VERSION = "10.2" ]; then
+if [ $CUDA_VERSION = "10.0" ]; then
+  CUBLAS=libcublas10
+  CUBLAS_DEV=libcublas-dev
+elif [ $CUDA_VERSION = "10.1" ]; then
+  # Have to pin to libcublas10=10.2.1.243-1 due to bug in TF, see
```

Comment:
I suspect this isn't true: we're mostly using the cublas installation in the docker container just for its headers, not for its library. We will dlopen() the library at run time I think.

That said, a pin does no harm.

Reply:
SG, let's keep it in sync with TF Dockerfile for now, which pins to this version

https://github.com/tensorflow/tensorflow/blob/3182f482fd7cc19fe5f75bf502f148c8583e3c18/tensorflow/tools/dockerfiles/dockerfiles/devel-gpu.Dockerfile#L44